buildpipeline/Core-Setup-Linux-BT.json

```diff
@@ -526,6 +526,96 @@
         "failOnStandardError": "false"
       }
     },
+	{
+      "enabled": true,
+      "continueOnError": false,
+      "alwaysRun": false,
+      "displayName": "Initialize docker - Debian 9",
```

Comment:
No,we are not rebuilding everything . Like other native installers, we consume prebuilt Linux-x64 portable binaries by volume mapping the prebuilt binaries and build native installers for each distro using corresponding docker container.